leftwm-core/src/handlers/command_handler.rs

```diff
@@ -691,6 +692,24 @@ fn focus_window_top(state: &mut State, swap: bool) -> Option<bool> {
     None
 }
 
+fn close_all_other_windows(state: &mut State) -> Option<bool> {
+    let current_window: Option<WindowHandle> = state.focus_manager.window(&state.windows).map(|w| w.handle);
+    let current_workspace = state.focus_manager.workspace(&state.workspaces);
+
+    if current_window.is_none() {
```

Comment:
Since we have an Option<bool> return type, we might consider just coercing this to return None like so:

```rs
fn close_all_other_windows(state: &mut State) -> Option<bool> {
    let current_window: Option<WindowHandle> =
        state.focus_manager.window(&state.windows).map(|w| w.handle);
    let current_workspace = state.focus_manager.workspace(&state.workspaces);

    for window in &state.windows {
        if window.handle.ne(&current_window?)
            && current_workspace?.is_displaying(&window)
        {
            let act = DisplayAction::KillWindow(window.handle);
            state.actions.push_back(act);
        }
    }
    Some(true)
}
```

This way we don't panic if something weird happens, and we can be a bit less verbose.